src/core/fresolver.cpp

```diff
@@ -21,10 +22,15 @@ bool FileResolver::contains(const fs::path &p) const {
 
 fs::path FileResolver::resolve(const fs::path &path) const {
     if (!path.is_absolute()) {
+        Log(Debug, "Looking for file \"%s\" on the resource search path(s)", path.native());
```

Comment:
Shouldn't this be logged into `Trace` as well?

Reply:
Currently `Debug` is written once per file lookup, and `Trace` is written once per path check. You're right that I forgot to paste the output--maybe it makes the behavior more clear.

Reply:
IMO this is a little too much for debug mode. I would like to be able to compile in debug mode while not always having those logs. Let's use `Trace` here to have control. In any case, the log messages you are adding in this PR are useful to debug a situation where a file couldn't be found, in which case you will need to switch to `Trace` to see all the messages.

Reply:
Ok, agree, I will make all of the outputs part of `Trace`.